rpcs3/Emu/RSX/RSXThread.cpp

```diff
@@ -563,7 +563,16 @@ namespace rsx
 		}
 
 		on_semaphore_acquire_wait();
-		std::this_thread::yield();
+
+		if ((state & (cpu_flag::dbg_global_pause + cpu_flag::exit)) == cpu_flag::dbg_global_pause)
+		{
+			// Wait 16ms during emulation pause. This reduces cpu load while still giving us the chance to render overlays.
+			thread_ctrl::wait_for(16000);
```

Comment:
```suggestion
			thread_ctrl::wait_on(state, old_state, 16000);
```
Name the argument "old_state".

Reply:
which would be?

Reply:
this?